Stripe/STPPaymentMethodEnums.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  STPPaymentMethodEnums.swift
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/12/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The type of the PaymentMethod.
+@objc public enum STPPaymentMethodType: Int {
+  /// A card payment method.
+  case card
+  /// An Alipay payment method.
+  case alipay
+  /// A GrabPay payment method.
+  case grabPay
+  /// An iDEAL payment method.
+  @objc(STPPaymentMethodTypeiDEAL) case iDEAL
+  /// An FPX payment method.
+  case FPX
+  /// A card present payment method.
+  case cardPresent
+  /// A SEPA Debit payment method.
+  @objc(STPPaymentMethodTypeSEPADebit) case SEPADebit
+  /// An AU BECS Debit payment method.
+  @objc(STPPaymentMethodTypeAUBECSDebit) case AUBECSDebit
+  /// A Bacs Debit payment method.
+  case bacsDebit
+  /// A giropay payment method.
+  case giropay
+  /// A Przelewy24 Debit payment method.
+  case przelewy24
+  /// An EPS payment method.
+  @objc(STPPaymentMethodTypeEPS) case EPS
+  /// A Bancontact payment method.
+  case bancontact
+  /// An OXXO payment method.
+  @objc(STPPaymentMethodTypeOXXO) case oxxo
```

Comment:
I wonder if this should be OXXO?

Reply:
I didn't want to change it, but we *just* launched OXXO so it's probably okay to flip.